src/components/ProteinsForDiseaseCardContainer.tsx

```diff
@@ -40,46 +54,76 @@ const ProteinsForDiseaseCardContainer: FunctionComponent<
     });
   };
 
-  if (!data) {
-    return null;
-  }
+  const handleFilterClick = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSelectedFilters({
+      ...selectedFilters,
+      [e.target.name]: e.target.checked,
+    });
+  };
 
-  const sortedData = data.results.sort(
-    (a: ProteinData, b: ProteinData) => (b.isExternallyMapped ? 1 : 0) && -1
+  const appliedFilters = Object.keys(selectedFilters).filter(
```

Comment:
To avoid `as unknown as` I'd go for
```
  const appliedFilters = (Object.keys(selectedFilters) as Array<
    keyof typeof selectedFilters
  >).filter((f) => selectedFilters[f]);
```

Reply:
Actually, also solves the following `as unknown as`